remote/gchat/helper.go

```diff
@@ -95,6 +94,7 @@ func toMessage(m *pubsub.Message) (models.Message, error) {
 	if event.User != nil {
 		message.Vars["_user.name"] = event.User.DisplayName
 		message.Vars["_user.id"] = event.User.Name
+		message.Vars["_user.internal_id"] = event.User.Name
```

Comment:
curious about this, since it's already available as `_user.id`?

Reply:
`_user.id` is overwritten in the following lines by the email, if it exists. https://github.com/target/flottbot/blob/1f7c870c45436213427acfbb13e19fae9a718476/remote/gchat/helper.go#L103

Reply:
Ah, thanks! I missed that, sorry. I wonder why we are not using `_user.email` for that other spot as we are using that for Slack. Seems a little cleaner.

Reply:
@justmiles friendly ping - was there are particular reason for overwriting `_user.id` with email? otherwise, i think populating `_user.email` would be more appropriate.

Reply:
@wass3r , I absolutely agree that `_user.email` looks much more logical and cleaner, I left it as is, adding a new field only in order not to break backward compatibility in existing user scripts. However, I am happy to rename the fields if you see no issues with backward compatibility.

Reply:
@vsychov i appreciate that you made the change in a thoughtful manner! i'll give it a day or so for chance of response here. otherwise, happy to merge this and might raise PR myself for making that breaking change and call it out in release notes.

Reply:
i'll go ahead and merge. thanks again.